public/app/plugins/panel/timeseries/config.ts

```diff
@@ -154,6 +154,18 @@ export function getGraphFieldConfig(cfg: GraphFieldConfig): SetFieldConfigOption
           shouldApply: (f) => f.type !== FieldType.time,
           process: identityOverrideProcessor,
         })
+        .addCustomEditor<void, boolean>({
+          id: 'insertNulls',
+          path: 'insertNulls',
+          name: 'Disconnect values',
```

Comment:
Connect + disconnect null values seems a little hard to understand - we should definitely include some documentation on this. Would be cool if we could include a little info button next to the field name that when hovered had more of an explanation, not sure if we currently have that ability or not - but even having a link to documentation here would be very helpful to end user on what the features are / how to use them (as a feature improvement)